app/views/measures/grouped/_table.html.erb

```diff
@@ -2,17 +2,21 @@
   <thead>
     <tr>
       <th>Country</th>
-      <th>Measure</th>
-      <th>Value</th>
-      <th>Conditions that apply</th>
+      <th>Measure type</th>
+
+      <% unless defined?(hide_duty_rate) && hide_duty_rate %>
```

Comment:
I am aware this is a "double negation",
BUT if we use the direct form, for example: `<% if defined?(show_duty_rate) && show_duty_rate %>`

We would need to pass in the local `show_duty_rate: true` in every place table are displayed.

Reply:
I tried, but it does not work, it returns nil even if hide_duty_rate is defined in the locals.
I guess `hide_duty_rate` is not considered a method, thus try(:hide_duty_rate) does not work.

Also, `try` is not used in any other views in the frontend, so I do not think that's a viable solution.

A different way would be `local_assigns`:
https://apidock.com/rails/v6.1.3.1/ActionView/Template/local_assigns

but I don't see any advantage on using defined?

Reply:
So I'd suggest either: 

1. Ask the measure we're rendering whether we should hide the duty rate (measures.shows_duty?) or;
2. Just use the same api when rendering every partial

Reply:
`defined?` take the name of the variable you want to check, I think if you want to check the existence of a method then you pass the symbol,
in our case, it's fine to pass the name of the local var, notice that it returns "local-variable" and not true/false.

Reply:
I am not sure if `measure` should know about the way it is going to be represented.
Not sure about "Just use the same api when rendering every partial",
you mean do not make the `hide_duty_rate` optional, so we can remove the `defined?`, I'd be happy with that, the downside is that we'd need to specify that param in every `render ".../table"` 

> So I'd suggest either:
> 
> Ask the measure we're rendering whether we should hide the duty rate (measure.displays_duty?) or;
> Just use the same api when rendering every partial